client/src/components/RejectForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import Button from "../components/Button";
+import TextAreaInput from "../components/TextAreaInput";
+
+const { REACT_APP_PROXY } = process.env;
+
+const RejectForm = (props) => {
+    const [coopRejected, setCoopRejected] = React.useState(false);
+    const [rejectReason, setRejectReason] = useState("");
+    const [approved, setApproved] = React.useState(false);
+    const [errors, setErrors, getErrors] = React.useState();
+    const [loadErrors, setLoadErrors] = React.useState("");
+
+    const rejRequest = async () => {
```

Comment:
Would you mind moving this function into the CoopService class, just to keep the organizational structure a little tidier?